docs/source/conf.py

```diff
@@ -41,6 +41,7 @@
     "_pytest": ("https://doc.pytest.org/en/latest/", None),
     # "PIL": ("http://pillow.readthedocs.org/en/latest/", None),
     "nbclient": ("https://nbclient.readthedocs.io/en/latest/", None),
+    "nbdime": ("https://nbdime.readthedocs.io/en/latest/", None),
```

Comment:
but now you should be able to remove the nbdime items in `nitpick_ignore` 😅

Reply:
Thanks. I just tried, but it doesn't work.

Reply:
Also tried adding `("py:class", "DiffConfig"): ("py:class", "nbdime.diffing.config.DiffConfig")` to `intersphinx_aliases`, but it does not make any difference either.